ngx-fudis/projects/ngx-fudis/src/lib/services/form/error-summary/internal-error-summary.service.ts

```diff
@@ -40,20 +40,18 @@ export class FudisInternalErrorSummaryService implements OnDestroy {
   }
 
   /**
-   * ------------------
-   *
    * CLASS VARIABLES
```

Comment:
It would be nice if these were more perceivable, so my suggestion is to use basic `//` commenting instead of JSDoc format. Including class variables, getters & setters ..., add and remove errors, reloading list..., dom related functions and misc headings.

For example:
```
  // --------------------
  //
  // CLASS VARAIBLES
  //
  // --------------------
```